### ISSUE TITLE
[enhancments] suggestions

### ISSUE THREAD

**Issue by splace** [vrml2/tnakgame#5]
modern default settings, (larger etc.)

damage as percentage

stay with explosion a bit longer

zero height border on terrain

wind as animated gif

**Comment by dwblaikie** [comment]
Yeah - I'd probably do damage as health bars, possibly over or beside the tanks themselves? (colour coded - green for high health, orange, red, etc)

Not sure I understand "zero height border on terrain" - could you describe that in more detail?

Wind as animated gif - *nod* perhaps, or a more organic shape at least - I like the volumetric nature of the arrow. Could go with squiggly lines, more lines for more wind?

**Comment by splace** [comment]
> Not sure I understand "zero height border on terrain" - could you describe that in more detail?

its just an idea about being able to see underneath the terrain, (with which, by the way, the black/unilluminated state seems to me to be the right situation for the lighting, maybe it originally looked different due to a browser peculiarity? ) by making the height at the edge go down to zero.

**Comment by splace** [comment]
> Wind as animated gif - nod perhaps, or a more organic shape at least - I like the volumetric nature of the arrow. Could go with squiggly lines, more lines for more wind?

left out detail there; i meant using texture transform to make it move, not actual animated gif, so you can change the speed/angle.(testing/demoing a feature as well.)

**Comment by dwblaikie** [comment]
> > Not sure I understand "zero height border on terrain" - could you describe that in more detail?
> 
> its just an idea about being able to see underneath the terrain, (with which, by the way, the black/unilluminated state seems to me to be the right situation for the lighting, 

Yeah, it's right that the underside is black/unilluminated - but the bug I saw a few weeks ago on mobile was the topside being black. But yeah, will have to test it again, etc.

> maybe it originally looked different due to a browser peculiarity? ) by making the height at the edge go down to zero.

Oh, fair, that's an option - I /think/ I like the way it is, but welcome to go in other directions/see how it looks :)

**Comment by dwblaikie** [comment]
> > Wind as animated gif - nod perhaps, or a more organic shape at least - I like the volumetric nature of the arrow. Could go with squiggly lines, more lines for more wind?
> 
> left out detail there; i meant using texture transform to make it move, not actual animated gif, so you can change the speed/angle.(testing/demoing a feature as well.)

Makes sense - I could see that. Certainly worth some experiments.

**Comment by splace** [comment]
setup options shortcuts.

**Comment by splace** [comment]
demo sequence
